install/kubernetes/helm/istio/charts/kiali/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
```

Comment:
What is kiali - shouldn't the description be about kiali ?

Reply:
Done